client/components/editor/Sidebar/Discussion.vue

```diff
@@ -1,19 +1,48 @@
 <template>
   <div class="my-4 pa-4">
     <discussion
-      @change="$emit('change', $event)"
       :activity="activity"
       show-notifications />
   </div>
 </template>
 
 <script>
+import { mapGetters, mapMutations } from 'vuex';
 import Discussion from '@/components/repository/common/Sidebar/Discussion/Embed';
+import get from 'lodash/get';
 
 export default {
   name: 'editor-discussion',
   props: {
-    activity: { type: Object, required: true }
+    activity: { type: Object, required: true },
+    isVisible: { type: Boolean, default: false }
+  },
+  computed: {
+    ...mapGetters('repository/comments', ['getActivityComments']),
+    comments: vm => vm.getActivityComments(vm.activity.id),
+    lastCommentAt: vm => new Date(get(vm.comments[0], 'createdAt', 0)).getTime()
+  },
+  methods: {
+    ...mapMutations('repository/comments', ['markSeenComments']),
+    setLastSeenComment(timeout) {
+      const latestComment = {
```

Comment:
Let's rename this one to `payload` (Since it does not represent `lastComment` model data) ;)